pkg/api/endpoints.go

```diff
@@ -18,17 +18,82 @@ package api
 
 import "github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/featureflags"
 
+const (
+	AlertingProfile                      = "alerting-profile"
+	NetworkZone                          = "network-zone"
+	ManagementZone                       = "management-zone"
+	Autotag                              = "auto-tag"
+	Dashboard                            = "dashboard"
+	DashboardV2                          = "dashboard-v2"
+	Notification                         = "notification"
+	Extension                            = "extension"
+	ExtensionElasticSearch               = "extension-elasticsearch"
+	CustomServiceJava                    = "custom-service-java"
+	CustomServiceDotnet                  = "custom-service-dotnet"
+	CustomServiceGo                      = "custom-service-go"
+	CustomServiceNodeJs                  = "custom-service-nodejs"
+	CustomServicePhp                     = "custom-service-php"
+	AnomalyDetectionMetrics              = "anomaly-detection-metrics"
+	AnomalyDetectionDisks                = "anomaly-detection-disks"
+	SyntheticLocation                    = "synthetic-location"
+	SyntheticMonitor                     = "synthetic-monitor"
+	ApplicationWeb                       = "application-web"
+	ApplicationMobile                    = "application-mobile"
+	AppDetectionRule                     = "app-detection-rule"
+	AwsCredentials                       = "aws-credentials"
+	KubernetesCredentials                = "kubernetes-credentials"
```

Comment:
Should we add some `// #nosec G101` comments to suppress the check failures for the credentials entries in the enum?